src/test/test-open-file.c

```diff
@@ -0,0 +1,33 @@
+
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <fcntl.h>
+
+#include "open-file.h"
+#include "string-util.h"
+#include "tests.h"
+
+TEST(parse_open_file) {
```

Comment:
No need to create a new test for `parse_open_file()`, testing it as part of testing `config_parse_open_file()` is sufficient.